swipe/src/main/kotlin/me/saket/swipe/SwipeableActionsBox.kt

```diff
@@ -52,6 +52,7 @@ fun SwipeableActionsBox(
   endActions: List<SwipeAction> = emptyList(),
   swipeThreshold: Dp = 40.dp,
   backgroundUntilSwipeThreshold: Color = Color.DarkGray,
+  limitSwipeDistanceToSwipeThreshold: Boolean = false,
```

Comment:
Using a threshold as the limit wouldn't be a good idea, they are serving opposite purposes. Let's add a `limitSwipeDistanceTo: Dp?` param instead?